cpp/ql/lib/experimental/semmle/code/cpp/dataflow/ProductFlow.qll

```diff
@@ -11,14 +11,31 @@ module ProductFlow {
      *
      * `source1` and `source2` must belong to the same callable.
      */
-    abstract predicate isSourcePair(DataFlow::Node source1, DataFlow::Node source2);
+    predicate isSourcePair(DataFlow::Node source1, DataFlow::Node source2) { none() }
+
+    predicate isSourcePair(
```

Comment:
Can you add QLdoc for this (and `isSinkPair/4`)?

Reply:
Done in dc00643ad100435ca6520397f4af3342da714074 👍.